Fixed a bunch of grammatical errors, as well as made some wordings more clear (#15)
osucord/resources: content.md

### Beatmap Tools
- [osu!Trainer](https://github.com/FunOrange/osu-trainer) - A program that allows you to modify the difficulty of a beatmap very quickly and easily
- [cosu!Trainer](https://github.com/hwsmm/cosutrainer) - osu!Trainer alternative for Linux, written in C

> - [Batch Beatmap Downloader](https://github.com/nzbasic/batch-beatmap-downloader) - Download a lot of beatmaps at once; you can also apply filters, such as year ranked, and create collections for the downloaded beatmaps.

Reviewer: ```suggestion
- [Batch Beatmap Downloader](https://github.com/nzbasic/batch-beatmap-downloader) - Tool to mass download beatmaps, you can also apply filters and create collections for the downloaded beatmaps.
```
not your fault just think this would be better

Reviewer 2: ```suggestion
- [Batch Beatmap Downloader](https://github.com/nzbasic/batch-beatmap-downloader) - Tool to mass download beatmaps, you can also apply filters and create collections for the downloaded beatmaps
```

---
### Browser Extensions
- [osuplus](https://osu.ppy.sh/community/forums/topics/408541?n=1) - Extension that adds extra features to the osu! website
- [score inspector](https://github.com/darkchii/score-inspector-extension) - A userscript that adds more stats to user profiles on the official osu! website (Extension compatible with the [osu! scores inspector](https://score.kirino.sh/))
- [osu! preview](https://github.com/JerryZhu99/osu-preview) - Preview a beatmap without downloading it
- [osu! subdivide nations](https://github.com/Cavitedev/osu-subdivide-nations) - Display the regional flags and names of players on the official osu! website as well as other unofficial sites (Extension compatible with [osu! World](https://osuworld.octo.moe/))
---
### Streaming

> - [StreamCompanion](https://github.com/Piotrekol/StreamCompanion) - Various overlays such as pp counter, map information, and more

Reviewer: ```suggestion
- [StreamCompanion](https://github.com/Piotrekol/StreamCompanion) - Provides various overlays such as pp counters, map information, and more. Can be used for streaming or in-game.
```

Reviewer 2: ```suggestion
- [StreamCompanion](https://github.com/Piotrekol/StreamCompanion) - Provides various overlays such as pp counters, map information, and more. Can be used for streaming or in-game
```
